src/main/java/mate/academy/Main.java

```diff
@@ -44,12 +52,31 @@ public static void main(String[] args) {
         yesterdayMovieSession.setMovie(fastAndFurious);
         yesterdayMovieSession.setShowTime(LocalDateTime.now().minusDays(1L));
 
-        MovieSessionService movieSessionService = null;
+        MovieSessionService movieSessionService = (MovieSessionService) injector.getInstance(
+                MovieSessionService.class);
         movieSessionService.add(tomorrowMovieSession);
         movieSessionService.add(yesterdayMovieSession);
 
         System.out.println(movieSessionService.get(yesterdayMovieSession.getId()));
         System.out.println(movieSessionService.findAvailableSessions(
                         fastAndFurious.getId(), LocalDate.now()));
+
+        AuthenticationService authenticationService = (AuthenticationService) injector.getInstance(
+                AuthenticationService.class);
+        try {
+            User registeredUser = authenticationService.register("email@gmail.com", "password");
+            System.out.println(registeredUser);
+        } catch (RegistrationException e) {
+            throw new RuntimeException("Failed to register a new user."
+                    + " Please check your email or password.");
+        }
+
+        try {
+            User loginedUser = authenticationService.login("email@gmail.com", "password");
+            System.out.println(loginedUser);
+        } catch (AuthenticationException e) {
+            throw new RuntimeException("Failed to login in account, "
+                    + "email or password is incorrect.");
```

Comment:
Similar to the previous comment, consider logging the exception details here as well. This will provide more information about the authentication failure, which can be useful for debugging.